pkg/consts/labels.go

```diff
@@ -18,4 +18,8 @@ const (
 	YTComponentLabelYqlAgent        string = "yt-yql-agent"
 	YTComponentLabelClient          string = "yt-client"
 	YTComponentLabelMasterCache     string = "yt-master-cache"
+	YTComponentLabelQueryTracker    string = "yt-query-tracker"
+	YTComponentLabelQueueAgent      string = "yt-queue-agent"
+	YTComponentLabelSPYT            string = "ytsaurus-spyt"
+	YTComponentLabelCHYT            string = "ytsaurus-chyt"
```

Comment:
Will something breaks (k8s labels?) if these are will be called `yt-spyt` and `yt-chyt` also?
Do we want to have them consistent with others or it is intended?

Reply:
well, I didn't dive into deep enough, it's just string constants moved out from component constructors to be aligned with the rest of component types